modules/collect/src/main/java/com/opengamma/strata/collect/MapStream.java

```diff
@@ -58,7 +58,7 @@
    * applying a function to each value.
    *
    * @param collection  the collection of values
-   * @param keyFunction  a function that returns the key for a value
+   * @param keyFunction  a function which returns the key for a value
```

Comment:
My reading of [grammar](http://www.kentlaw.edu/academics/lrw/grinker/LwtaThat_Versus_Which.htm) [rules](http://www.quickanddirtytips.com/education/grammar/which-versus-that-0) suggests "that" is correct here.;-)